index.html

```diff
@@ -46,22 +45,19 @@ <h2 class="desktop">Hi there. I'm Alfred</h2>
                 <li class="Angle social"><a href="#"><img src="images/angellist.svg" alt="Angle"></a></li>
             </ul>
             <div class="drop-down"> <img src="images/dropdown.png" alt=""></div>
-
         </section>
         <!-- card one -->
         <section class="top-card-one">
             <h2>My Recent <br> Works</h2>
             <div class="bd">
                 <img src="images/ImgPlaholder.png" alt="">
                 <h2>Multi-Post Stories <br> Gain+Glory </h2>
-                <nav>
-                    <ul class="technologies">
-                        <li class="tech"><a href="">Ruby on rails</a></li>
-                        <li class="tech"><a href="">css </a></li>
-                        <li class="tech"><a href=""> java script </a></li>
-                        <li class="tech"><a href=""> html </a></li>
-                    </ul>
-                </nav>
+                <ul class="technologies">
+                    <li class="tech"><a href="">Ruby on rails</a></li>
+                    <li class="tech"><a href="">css </a></li>
+                    <li class="tech"><a href=""> java script </a></li>
+                    <li class="tech"><a href=""> html </a></li>
+                </ul>
```

Comment:
- [ ] [OPTIONAL] You could wrap each card with a `<div>` tag instead of a `<section>` tag. Use `<section>` tags to wrap whole sections.